src/Core/src/Handlers/Editor/EditorHandler.Android.cs

```diff
@@ -123,16 +121,6 @@ static void MapFocus(IEditorHandler handler, IEditor editor, object? args)
 				handler.PlatformView.Focus(request);
 		}
 
-		void OnPlatformViewAttachedToWindow(object? sender, ViewAttachedToWindowEventArgs e)
```

Comment:
A similar change has just been merged here:
https://github.com/dotnet/maui/pull/26101/files#diff-04130ba6aadaa5a68624b127e29d3f85384872fc0520ed02821a5b96ac78ecbb

Reply:
For Entry, this is Editor, thats why I opened this :)

Reply:
Ohhh, right!